packages/website/webpack.config.js

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable import/no-extraneous-dependencies */
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
+  entry: './client.js',
+  // output : {
```

Comment:
if these commented code sections are here on purpose, could you please add a comment at the top to explain them and give instructions to anybody using this?